tracing/trace_state.hh

```diff
@@ -501,6 +501,7 @@ private:
     friend void add_prepared_statement(const trace_state_ptr& p, prepared_checked_weak_ptr& prepared);
     friend void set_username(const trace_state_ptr& p, const std::optional<auth::authenticated_user>& user);
     friend void add_table_name(const trace_state_ptr& p, const sstring& ks_name, const sstring& cf_name);
+    friend void add_alternator_table_name(const trace_state_ptr& p, const sstring& cf_name);
```

Comment:
Please use `std::string_view`, not `const sstring&`. This is always good advice, but even more so in Alternator which often uses std::string, **not** sstring, so by using `sstring &` you are forcing the compiler to make a copy.